src/install/browserFetcher.ts

```diff
@@ -73,6 +74,7 @@ function getDownloadUrl(browserName: BrowserName, platform: BrowserPlatform): st
       ['mac10.13', '%s/builds/firefox/%s/firefox-mac.zip'],
       ['mac10.14', '%s/builds/firefox/%s/firefox-mac.zip'],
       ['mac10.15', '%s/builds/firefox/%s/firefox-mac.zip'],
+      ['mac10.16', '%s/builds/firefox/%s/firefox-mac.zip'],
```

Comment:
I don't have a Big Sur machine at hand. Does our current binary just work on 10.16?

Reply:
it does. just upgraded my dev machine to big sur yesterday.

which is why i opened this pr.